cms/djangoapps/contentstore/management/commands/swap_from_auto_track_cohort_pilot.py

```diff
@@ -0,0 +1,211 @@
+from contentstore.course_group_config import GroupConfiguration
+from contentstore.models import MigrateVerifiedTrackCohortsSetting
+from django.conf import settings
+from course_modes.models import CourseMode
+from django.core.management.base import BaseCommand, CommandError
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+
+from openedx.core.djangoapps.course_groups.cohorts import CourseCohort
+from openedx.core.djangoapps.course_groups.models import (CourseUserGroup, CourseUserGroupPartitionGroup)
+from openedx.core.djangoapps.verified_track_content.models import VerifiedTrackCohortedCourse
+
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.django import modulestore
+from xmodule.partitions.partitions import ENROLLMENT_TRACK_PARTITION_ID
+from xmodule.partitions.partitions_service import PartitionService
+
+
+class Command(BaseCommand):
+    """
+    Migrates a course's xblock's group_access from Verified Track Cohorts to Enrollment Tracks
+    """
+
+    def handle(self, *args, **options):
+
+        errors = []
+
+        module_store = modulestore()
+
+        verified_track_cohorts_settings = self._enabled_settings()
+
+        for verified_track_cohorts_setting in verified_track_cohorts_settings:
+            old_course_id = verified_track_cohorts_setting.old_course_id
+            rerun_course_id = verified_track_cohorts_setting.rerun_course_id
+            audit_cohort_names = verified_track_cohorts_setting.get_audit_cohort_names()
+
+            try:
+                old_course_key = CourseKey.from_string(old_course_id)
+            except InvalidKeyError:
+                try:
+                    old_course_key = SlashSeparatedCourseKey.from_string(old_course_id)
+                except InvalidKeyError:
+                    errors.append("Invalid course_key: '%s'" % old_course_id)
```

Comment:
I thought the request was to collect the errors to reduce number of iterations required for a user. I don't know if there's an alternative between continuing to execute and collecting the errors while not updating anything in modulestore, or just raising the exceptions as they're encountered. Do you have a preference or suggestions?